app/backend/lib/s3client.ts

```diff
@@ -25,3 +31,18 @@ aws.config.update({
 const s3Client = new aws.S3();
 
 export default s3Client;
```

Comment:
tech debt on leaving the v2 sdk for other processes needs to be tracked

Reply:
https://app.zenhub.com/workspaces/ccbc---delivery-board-61c0c1204c8bcb001491c5f3/issues/bcgov/conn-ccbc-portal/981